packages/acpica.rb

```diff
@@ -3,27 +3,27 @@
 class Acpica < Package
```

Comment:
Use Autotools instead?

Reply:
Autotools require a `configure` file, but this package only has a Makefile

Reply:
Gotcha